### ISSUE TITLE
Improve usability on touch devices

### ISSUE THREAD

**Issue by qulle** [qulle/oltb#7]
On some clickable items a blue highlighted background/selection is show. This is annoying. This has nothing to do with accessibility so it should be safe to disable.
